content/docs/rtl-support/index.md

```diff
@@ -0,0 +1,114 @@
+# RTL support
+The goal is to have full RTL support for all of ionic.
+If you don't need RTL support, you can set `$include-rtl: false`, and your css bundle will be smaller.
```

Comment:
I think that it should default to true, as beginners will probably won't know it, and will have a hard time with RTL, and then the more the user is advanced, the more likely it is for them to know performance tips for ionic, including disabling rtl if needed.

RTL is not disabled on iOS/Android apps by default, I think, because on iOS RTL version, I know that many apps just show the other way without coders intervention (I definitely might be wrong, just from my own experience)